src/openrct2-ui/interface/Window.cpp

```diff
@@ -536,6 +537,11 @@ void window_all_wheel_input()
         }
     }
 }
+void toggle_screensaver_lock()
```

Comment:
All new functions are to be TitleCase. (Really we need to start a big renaming project to make everything use TitleCase)